README.md

```diff
@@ -52,13 +52,13 @@ Please refer to the [License file](./LICENSE).
 ## Acknowledgments
 
 The [Otter Team](https://github.com/orgs/amadeus-digital/teams/otter) is responsible for the review of the code of this repository.
-Any bug of feature request can be addressed via [issue](https://github.com/AmadeusITGroup/otter/issues/new/choose) report.
+Any bug of feature request can be addressed via [issue](https://github.com/amadeus-digital/Otter/issues/new/choose) report.
```

Comment:
that's the link for the private repository

Reply:
I just fixed this, could you check the updated link that references the Otter team on line 54?